heat/core/operations.py

```diff
@@ -197,10 +200,12 @@ def local_argmin(*args, **kwargs):
         return torch.cat([minimums.double(), indices.double()])
 
     # perform the global reduction
-    reduced_result = __reduce_op(x, local_argmin, MPI_ARGMIN, axis, out)
+    reduced_result = __reduce_op(
+        x, local_argmin, MPI_ARGMIN, axis, keepdim, out)
```

Comment:
We should configure your IDE to not break these lines :D

Reply:
its probably in the setting under the name auto wrap or auto break or something like that

Reply:
:D @Markus-Goetz alright, We (pluralis majestatis) have configured now.

Reply:
Great! (I laughed my ass off here about the pluralis majestatis). Thanks Claudia!